test/run_test.py

```diff
@@ -186,6 +188,219 @@ def __contains__(self, item):
     "test_transformers",
 ]
 
+S390X_BLOCKLIST = [
+    # following tests fail for various reasons
+    "test_multiprocessing",
+    "ao/sparsity/test_composability",
+    "ao/sparsity/test_qlinear_packed_params",
+    "cpp/aot_inductor/test",
+    "cpp_api_parity/functional_impl_check",
+    "cpp_api_parity/module_impl_check",
+    "cpp_extensions/no_python_abi_suffix_test/setup",
+    "cpp_extensions/setup",
+    "custom_backend/test_custom_backend",
+    "custom_operator/test_custom_ops",
+    "distributions/test_distributions",
+    "dynamo/test_exc",
+    "dynamo/test_functions",
+    "dynamo/test_misc",
+    "dynamo/test_trace_rules",
+    "error_messages/storage",
+    "functorch/discover_coverage",
+    "functorch/xfail_suggester",
+    "fx/test_source_matcher_utils",
+    "fx/test_z3_gradual_types",
+    "inductor/minifier_smoke",
+    "inductor/test_aot_inductor",
+    "inductor/test_benchmark_fusion",
+    "inductor/test_cpu_repro",
+    "inductor/test_cudacodecache",
+    "inductor/test_custom_post_grad_passes",
+    "inductor/test_fused_attention",
+    "inductor/test_inductor_utils",
+    "inductor/test_inplacing_pass",
+    "inductor/test_kernel_benchmark",
+    "inductor/test_layout_optim",
+    "inductor/test_max_autotune",
+    "inductor/test_mkldnn_pattern_matcher",
+    "inductor/test_move_constructors_to_cuda",
+    "inductor/test_multi_kernel",
+    "inductor/test_pattern_matcher",
+    "inductor/test_perf",
+    "inductor/test_select_algorithm",
+    "inductor/test_snode_runtime",
+    "inductor/test_split_cat_fx_passes",
+    "inductor/test_triton_wrapper",
+    "inductor/test_unbacked_symints",
+    "jit/fixtures_srcs/generate_models",
+    "jit/fixtures_srcs/test_upgrader_models_generation",
+    "jit/test_autodiff",
+    "jit/test_backends",
+    "jit/test_complexity",
+    "jit/test_exception",
+    "jit/test_ivalue",
+    "jit/test_list_dict",
+    "jit/test_peephole",
+    "jit/test_script_profile",
+    "jit/test_tracer",
+    "jit/test_type_sharing",
+    "jit/test_types",
+    "jit/test_with",
+    "jit/xnnpack/test_xnnpack_delegate",
+    "lazy/test_meta_kernel",
+    "mobile/custom_build/prepare_model",
+    "mobile/model_test/gen_test_model",
+    "mobile/model_test/torchvision_models",
+    "mobile/model_test/update_production_ops",
+    "mobile/test_lite_script_module",
+    "mobile/test_lite_script_type",
+    "mobile/test_quantize_fx_lite_script_module",
+    "onnx/debug_embed_params",
```

Comment:
Looks like S390 does not support any ONNX or say inductor test. In that case, please use globs.

Reply:
There are a lot of currently skipped inductor tests, but it's still less than half of them. I'll double check onnx cases, since recently some onnx-related dependencies were added, and if still all fails, I'll use glob.